.travis.test

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+set -ex
+
+# enable coprs
+dnf install -y dnf-plugins-core
+dnf copr enable -y npmccallum/jansson
+# dnf copr enable -y npmccallum/jose
+
+# install build dependencies
+dnf install -y \
+    gcc redhat-rpm-config make pkgconfig \
+    findutils tar bzip2 \
+    jansson-devel openssl-devel zlib-devel \
+    python-tox python-pip Cython
+
+# dnf install -y jose libjose-devel libjose-openssl-devel libjose-zlib-devel
+
+# install Python dependencies
+case "$1" in
+  python2)
+    dnf install -y python-devel python-setuptools python-virtualenv
+    ;;
+  python3)
+    dnf install -y python3-Cython python3-devel python3-setuptools python3-tox python3-virtualenv
+    ;;
+  *)
+    echo "$0 python2|python3"
+    exit 1
+    ;;
+esac
+
+# install jose
```

Comment:
Let's not install from sources. Part of test infrastructure here is testing the package builds too. I'd suggest enabling my copr above. Once the development pace slows down a bit, we'll move to the stock Fedora packages.